src/test/scala-2/chiselTests/FormalContractSpec.scala

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chiselTests
+
+import chisel3._
+import chisel3.ltl._
+import chisel3.testers.BasicTester
+import chisel3.experimental.SourceLine
+import _root_.circt.stage.ChiselStage
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+import Sequence._
+
+class FormalContractSpec extends AnyFlatSpec with Matchers with FileCheck {
+  it should "support contracts with no operands" in {
```

Comment:
Nit: Usually a "Flat Spec" will have some "behavior of ???" lines to help structure it. Or it will have a leading:

``` scala
"Formal Contracts" should "foo" in { ??? }

they should "bar" in { ??? }

"Formal Contracts error behavior" should "baz" in { ??? }
```

This is very nitty, but it helps with readability of what the test is doing both when reading the code and when reading what Scalatest shows as output.

Reply:
Good point. I've added a `behavior of` line to this.